schemas/dataSources.properties.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "id": "https://immersive-web/gamepad-mapping/0.1.0/dataSources.properties.schema.json",
+    "type": "object",
+    "additionalProperties": false,
+    "definitions": {
+        "id": {
+            "description" : "An identifier for the data source",
+            "type" : "string"
+        },
+        "button": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["gamepadButtonsIndex"],
+            "properties": {
+                "gamepadButtonsIndex": { 
+                    "allOf": [{ "$ref": "gamepadIndex.schema.json" }]
+                },
+                "supportsTouch" : {
+                    "description" : "Indicates all of the buttons is capable of reporting GamepadButton.touched",
+                    "type" : "boolean",
+                    "default" : "false"
+                },
+                "supportsPress" : {
+                    "description" : "Indicates all of the buttons is capable of reporting GamepadButton.pressed",
+                    "type" : "boolean",
+                    "default" : "true"
+                },
+                "analogValues" : {
+                    "description" : "Indicates at least one of the buttons are capable of reporting numbers in between min and max",
+                    "type" : "boolean",
+                    "default" : "false"
+                }
+            }
+        },
+        "axis": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["gamepadAxesIndex"],
+            "properties": {
+                "gamepadAxesIndex": { 
```

Comment:
Is there anything else that you can see being placed in here at some point? Honestly it feels like this should simply be a `gamepadIndex`, rather than an object with a single (kind of verbose) key that points to a `gamepadIndex`.

Reply:
Yep!  The original idea is that we could put optional min/max in there, but then I added the concept of the browser overrides and I realized it might be easiest to just mimic the format in the section below... I'm open to other approaches though...

Reply:
And yes, I agree that it should be `gamepadIndex` instead of `gamepadButtonsIndex` and `gamepadAxesIndex`

Reply:
decided the dpad edge case regarding button properties being inconsistent can be handled in the overrides file